package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "near-lake-framework",
-  "version": "1.0.6",
+  "version": "2.0.0",
```

Comment:
Bumped the major version but I'm not actually sure this considered a breaking change? 🤔

Reply:
Unfortunately, it is a breaking change. Since the previous version is not going to get the `DelegatedAction` action when it is in the JSON